crates/swc_error_reporters/src/lib.rs

```diff
@@ -266,8 +266,8 @@ impl fmt::Display for MietteDiagnostic<'_> {
 
 fn convert_span(span: Span) -> SourceSpan {
     let len = span.hi - span.lo;
-    let start = SourceOffset::from(span.lo.0 as usize);
-    SourceSpan::new(start, SourceOffset::from(len.0 as usize))
+    let start = SourceOffset::from(span.lo.0 as usize + 1);
```

Comment:
Is this correct?

Reply:
seems like should `+1` here:

<img width="724" alt="image" src="https://github.com/swc-project/swc/assets/3468483/d961c9ec-2b5b-4a28-a88b-ae224c3c5f98">

![image](https://github.com/swc-project/swc/assets/3468483/85d47bb2-7c8a-4096-b220-fa174b9384a0)